<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrossroadXa</class>
 <widget class="QWidget" name="CrossroadXa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1160</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1160</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Crossroad Xa</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(186, 189, 182);</string>
  </property>
  <widget class="Line" name="separationMiddle">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>-2</y>
     <width>10</width>
     <height>515</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_1">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>156</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_2">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>255</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_3">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>355</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>15</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_8">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>15</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_9">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>15</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_4">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>354</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_5">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>254</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_6">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>155</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_10">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_12">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>360</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_13">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>17</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_14">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>362</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_15">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>357</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_16">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>257</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_17">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>17</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_18">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>359</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_19">
   <property name="geometry">
    <rect>
     <x>966</x>
     <y>356</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_20">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>17</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_21">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>158</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_22">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>362</y>
     <width>8</width>
     <height>145</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_23">
   <property name="geometry">
    <rect>
     <x>966</x>
     <y>256</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_24">
   <property name="geometry">
    <rect>
     <x>966</x>
     <y>157</y>
     <width>145</width>
     <height>8</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>0</y>
     <width>131</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Smart semaphore &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>0</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Conventional semaphore&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>80</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Cars passed: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalCarsSmart">
   <property name="geometry">
    <rect>
     <x>482</x>
     <y>80</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>978</x>
     <y>90</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Cars passed: &lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalCarsConv">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>90</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="separationLine_25">
   <property name="geometry">
    <rect>
     <x>232</x>
     <y>15</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_26">
   <property name="geometry">
    <rect>
     <x>332</x>
     <y>360</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_27">
   <property name="geometry">
    <rect>
     <x>812</x>
     <y>17</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_28">
   <property name="geometry">
    <rect>
     <x>912</x>
     <y>363</y>
     <width>5</width>
     <height>145</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_29">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>309</y>
     <width>145</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_30">
   <property name="geometry">
    <rect>
     <x>386</x>
     <y>207</y>
     <width>145</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_31">
   <property name="geometry">
    <rect>
     <x>616</x>
     <y>312</y>
     <width>145</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="separationLine_32">
   <property name="geometry">
    <rect>
     <x>966</x>
     <y>210</y>
     <width>145</width>
     <height>5</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="startSimulation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Start simulation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>0</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>A</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>0</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>S</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>170</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>I</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>K</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC5">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>D</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>J</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonC8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>U</string>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_1_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>125</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_1_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_1_1">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>130</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_1_2">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>364</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_3_1">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_3_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_3_1">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>130</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_3_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>360</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_2_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_2_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>363</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_2_1">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>140</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_2_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>365</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_4_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>263</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semS_4_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_4_1">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>265</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="Semaphore" name="semC_4_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>240</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Semaphore</class>
   <extends>QFrame</extends>
   <header>semaphore.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
